sycl/test-e2e/USM/lit.local.cfg

```diff
@@ -0,0 +1,4 @@
+import platform
```

Comment:
Why do we need `import platform`? Looking at [MemorySanitizer/lit.local.cfg ](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/MemorySanitizer/lit.local.cfg), I don't think it's necessary.

Reply:
Don't know, it's just a cherry-pick

Reply:
Should I delete it?

Reply:
> Should I delete it?

I've opened https://github.com/intel/llvm/pull/16594 to remove this line from `sycl` branch. If CI passes, yes, we can remove this line from this PR as well.